app-text/texlive/texlive-2019.recipe

```diff
@@ -7,20 +7,20 @@ HOMEPAGE="http://tug.org/texlive/"
 COPYRIGHT="1996 by collaboration between the TeX user groups."
 LICENSE="GNU GPL v2"
 REVISION="2"
-SOURCE_URI="ftp://tug.org/historic/systems/texlive/2014/texlive-20140525-source.tar.xz"
-SOURCE_URI_2="ftp://tug.org/historic/systems/texlive/2014/texlive-20140525-texmf.tar.xz"
-CHECKSUM_SHA256="08bb4eb02923ac65ce1183612c6b64da185904f26a1660f5ca4f902d055795be"
-CHECKSUM_SHA256_2="e0227251c5aa23f8b88973c298cc80b03bd238c645f7f82a6ce888a06b295963"
-SOURCE_DIR="texlive-20140525-source"
-SOURCE_DIR_2="texlive-20140525-texmf"
-PATCHES="texlive-2014.patchset"
+SOURCE_URI="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-20190410-source.tar.xz"
+SOURCE_URI_2="ftp://tug.org/historic/systems/texlive/$portVersion/texlive-20190410-texmf.tar.xz"
+CHECKSUM_SHA256="d2a29fef04e34dc3d2d2296c18995fc357aa7625e7a6bbf40fb92d83d3d0d7b5"
+CHECKSUM_SHA256_2="c2ec974abc98b91995969e7871a0b56dbc80dd8508113ffcff6923e912c4c402"
+SOURCE_DIR="texlive-20190410-source"
+SOURCE_DIR_2="texlive-20190410-texmf"
+PATCHES="texlive-2019.patchset"
 ADDITIONAL_FILES="tlutils.pm.patch"
 
 ARCHITECTURES="x86 !x86_gcc2 x86_64"
```

Comment:
I don't think texlive builds on x86_64 yet. But we should probably disable this unless you plan on doing this later in this PR. But your choice if you decide to do this later.

Reply:
I can't say it for sure. I checked the files where i386-haiku was added (see patches), but it doesn't mentions any x86_64 platforms. Maybe they are defined in some other file which needs to be patched aswell. 
I am still working on this, so far it built everything (?), but the install phase bails out telling me there is no binary for this platform.